src/features/Workout/SectionDetail.js

```diff
@@ -346,21 +348,51 @@ const SectionDetail = () => {
                   </span>
                   <span className="text-sm">Weight Choosing Guide</span>
                 </div>
-                <div>
-                  <h2 className="workout-gradient-text my-4 text-2xl">
-                    Data Inputs
-                  </h2>
-                  {dataInput.map((input, index) => (
-                    <DataInputComponent2
-                      key={index}
-                      inputId={input.id}
-                      inputType={input.type}
-                      inputOptions={input.options}
-                      placeholder={input.label}
-                      label={input.label}
-                    />
-                  ))}
-                </div>
+
+               <div>
+        <h2 className="workout-gradient-text my-4 text-2xl">
+          Data Inputs
+        </h2>
+        {dataInput.slice(0, 2).map((input, index) => (
```

Comment:
why is slice being used here?

Reply:
Slice is used so the first two data inputs are in a separate section and the third one is under Max effort section.